android/src/test/java/com/oblador/keychain/KeychainModuleTests.java

```diff
@@ -448,6 +450,11 @@ public void testDowngradeBiometricToAes_api28() throws Exception {
     final JavaOnlyMap options = new JavaOnlyMap();
     options.putString(Maps.SERVICE, "dummy");
 
+    final JavaOnlyMap promptOptions = new JavaOnlyMap();
+    promptOptions.putString(KeychainModule.AuthPromptOptions.TITLE, "Title");
```

Comment:
This test was failing because `PromptInfo` requires title and negativeText to be set even when `setUserConfirmationRequired(false)`.